config/initializers/rack_attack.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Rack::Attack
-  Rack::Attack.cache.store = Rack::Attack::StoreProxy::RedisStoreProxy.new(Redis.current)
+  Rack::Attack.cache.store = Rack::Attack::StoreProxy::RedisStoreProxy.new(Redis.new(REDIS_CONFIG[:redis]))
```

Comment:
This data will regenerate itself, so it's not worth the trouble to migrate it in real time. Instead, we can just flip it over to the new Redis whenever we want and let the old data die.

Reply:
Agreed 👍

Reply:
What's the trouble of migrating it?  When `RedisDuplicator` is pulled out, we'll want these to be `Redis.current` again.  So if it's just the additional monetary cost of unnecessary duplication, can we just let it ride until we're ready to remove `RedisDuplicator`?  I like using a hack to duplicate redis, but I don't like making two places that need to also be reverted.

Reply:
In `RedisDuplicator` we're only overriding `set`, `del`, & `expire` because that's all we use in `vets-api` to "save-and-remove" data in Redis. OTOH, `breakers`, uses `[zrem, zadd, incr, expire]`. OTOOH, I haven't yet bothered looking into which Redis calls `Rack::Attack::StoreProxy::RedisStoreProxy` uses.

I could try to encompass breakers and rack attack stuff, but I think that `RedisDuplicator` would get more complex, when really we care about user data a lot more than the utility data of breakers and rack attack.

Reply:
Didn't want to leave rack attack using `Redis.current` with the way `RedisDuplicator` is coded right now, since rack attack might be using `del`, `set,` and/or `expire`, which would leave us with some partial data state.

Reply:
That all makes sense.  If the "remove duplicator" ticket (whatever that is) could have a note about these places that aren't using `Redis.current`, that'd be nice.  Thanks for answering

Reply:
Donezo - https://github.com/department-of-veterans-affairs/va.gov-team/issues/6865#issuecomment-626831951